public/wp-content/themes/ColoredCow/header.php

```diff
@@ -6,8 +6,24 @@
     <meta charset="utf-8">
     <meta name="viewport" content="initial-scale=1">
     <title><?php echo bloginfo('name'); ?></title>
-    <?php wp_head(); ?>
+    <?php wp_head();?>
 </head>
 
+	<nav class=" navbar fixed-top navbar-toggleable-md navbar-inverse bg-inverse">				
+		<button class="navbar-toggler navbar-toggler-right" type="button" data-toggle="collapse" data-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
+			<span class="navbar-toggler-icon"></span>
+		</button>
+		<a class="navbar-brand" href="#" style="font-family: ;"><img src="<?php  $image_url=get_template_directory_uri()."/dist/img/logo.png"; echo esc_url($image_url);?>" width="180" height="45" class="d-inline-block align-top" alt=""></a>
```

Comment:
you can directly echo the complete URL without creating a variable. Also, it's not preferred to have multiple statements in an inline PHP tag.